swap_meet/vendor.py

```diff
@@ -1,2 +1,125 @@
+from swap_meet.item import Item
+from swap_meet.decor import Decor
+from swap_meet.clothing import Clothing
+from swap_meet.electronics import Electronics
+
 class Vendor:
-    pass
\ No newline at end of file
+
+    """
+    Wave 01: Represents a vendor managing an inventory of items.
+    Attributes: inventory (list), default is an empty list.
+    Methods: add(item) adds and returns the item; 
+    remove(item) removes and returns the item or False if not found.
+    """
+
+    def __init__(self, inventory=None):
+        if not inventory:
+            self.inventory = []
+        else:
+            self.inventory = inventory
+
+    def add(self,item):
+        self.inventory.append(item)
+        return item
+    
+    def remove(self,item):
+        if item in self.inventory:
+            self.inventory.remove(item)
+            return item
+        return False
```

Comment:
I'd consider flipping the logic so that your 'main' logic on lines 27 and 28 are not inside a conditional if block, but in the main level of indentation. 

Then you'd be able to use a guard clause to check if an item is not in inventory and immediately return False 

```suggestion
        if item not in self.inventory:
            return False
        self.inventory.remove(item)
        return item
```